spec/models/reports/facility_state_spec.rb

```diff
@@ -67,4 +59,103 @@
+        create(:patient, assigned_facility: facility, recorded_at: june_2021[:now] - 2.years)
+        create(:patient, assigned_facility: facility, recorded_at: june_2021[:under_12_months_ago])
+        create(:patient, assigned_facility: facility, recorded_at: june_2021[:under_3_months_ago])
+        create(:patient, assigned_facility: facility, recorded_at: june_2021[:end_of_month])
+
+        RefreshMaterializedViews.new.refresh_v2
+        with_reporting_time_zone do
+          expect(described_class.find_by(facility: facility, month_date: june_2021[:now] - 2.years).cumulative_assigned_patients).to eq 1
+          expect(described_class.find_by(facility: facility, month_date: june_2021[:under_12_months_ago]).cumulative_assigned_patients).to eq 2
+          expect(described_class.find_by(facility: facility, month_date: june_2021[:under_3_months_ago]).cumulative_assigned_patients).to eq 3
+          expect(described_class.find_by(facility: facility, month_date: june_2021[:now]).cumulative_assigned_patients).to eq 4
+        end
+      end
+    end
+  end
+
+  context "treatment outcomes in the last 3 months" do
+    it "computes totals for under care patients" do
+      facility = create(:facility)
+
+      patient_controlled = create(:patient, assigned_facility: facility, recorded_at: june_2021[:long_ago])
+      create(:bp_with_encounter, :under_control, patient: patient_controlled, recorded_at: june_2021[:now] - 1.month)
+
+      patient_uncontrolled = create(:patient, assigned_facility: facility, recorded_at: june_2021[:long_ago])
+      create(:bp_with_encounter, :hypertensive, patient: patient_uncontrolled, recorded_at: june_2021[:now] - 1.months)
+
+      patient_missed_visit = create(:patient, assigned_facility: facility, recorded_at: june_2021[:long_ago])
+      create(:bp_with_encounter, patient: patient_missed_visit, recorded_at: june_2021[:over_3_months_ago])
+
+      patient_visited_no_bp = create(:patient, assigned_facility: facility, recorded_at: june_2021[:long_ago])
+      create(:prescription_drug,
+        device_created_at: june_2021[:now] - 1.month,
+        facility: facility,
+        patient: patient_visited_no_bp,
+        user: patient_visited_no_bp.registration_user)
+      create(:blood_pressure, patient: patient_visited_no_bp, recorded_at: june_2021[:over_3_months_ago])
+
+      RefreshMaterializedViews.new.refresh_v2
+      with_reporting_time_zone do
+        facility_state_june_2021 = described_class.find_by(facility: facility, month_date: june_2021[:now])
+
+        expect(facility_state_june_2021.controlled_under_care).to eq 1
+        expect(facility_state_june_2021.uncontrolled_under_care).to eq 1
+        expect(facility_state_june_2021.missed_visit_under_care).to eq 1
+        expect(facility_state_june_2021.visited_no_bp_under_care).to eq 1
+        expect(facility_state_june_2021.patients_under_care).to eq 4
+        expect(facility_state_june_2021.patients_lost_to_follow_up).to eq 0
```

Comment:
We should probably have better names for these 2 fields. We currently have
- `under_care`: number of assigned patients by care state
- `patients_under_care`: **adjusted** number of assigned patients by care state

Should we perhaps prefix all the adjusted fields with `adjusted_`?

Reply:
We definitely should. I'm not a big fan of the `adjusted` term. Ideally, I'd love to use a term like that means "eligible for control", or "without new people", or "controllable", or something. If we can't find the right word for that, then `adjusted` it is.

Reply:
I think we should stick with `adjusted_` for now because it is the term we use all throughout the code base currently. Introducing a new term for this in the pipeline _without_ changing it everywhere in the code will lead to lots of confusion.

I'm open to any suggestions for better terms going forward, but we if we do come up with one we should do that rename across the app and the DB to avoid fragmentation.  FWIW, @claudiovallejo and I tried a few times to come up with a better term for this, and adjusted was the best we could come up with 🤷‍♂️ . We don't have a succinct term for it in the actual reports UI, which I think reflects how hard naming this is.

Reply:
Can we continue this conversation with https://github.com/simpledotorg/simple-server/pull/2754/files#r677140960? There's some more context there on naming, and I'm doing some renames there anyway.

Reply:
Good to know we've thought about this loads already. Cool, in the absence of a better succinct term that we can propagate everywhere, let's stick to `adjusted` for now.

Reply:
And yes, let's continue this conversation in the other PR 👍